mito-ai/src/Extensions/AiChat/PromptManager.tsx

```diff
@@ -7,7 +7,9 @@ export function createBasicPrompt(
 ): string {
     const prompt = `You are an expert python programmer writing a script in a Jupyter notebook. You are given a set of variables, existing code, and a task.
 
-Respond with the updated active code cell and a short explanation of the changes you made.
+There are two possible types of responses you might give:
+1. Code Update: If the task requires modifying or extending the existing code, respond with the updated active code cell and a short explanation of the changes made. 
+2. Explanation/Information: If the task does not require a code update, provide an explanation, additional information about a package, method, or general programming question, without writing any code.
```

Comment:
Maybe we want to throw the word "concise" into the second one. A few of the responses I'm getting a quite long. "Provide a concise explanation"

Reply:
Good catch. Added a sentence to the end of the second instruction. Dropped the character count of the response by ~30%. 

<img width="621" alt="Screenshot 2024-12-11 at 10 29 53 AM" src="https://github.com/user-attachments/assets/4fc09df1-d992-4201-bf90-bd985abc28e4" />